tests/unit/UnitTests.d

```diff
@@ -932,3 +932,21 @@ void bar (const(foo_t)* foo);
 D");
 
 }
+
+// Octal unit tests.
+unittest
+{
+    assertTranslates(
+    q"C
+#define some_octal_number 0101
+#define zero_is_not_octal 0
+#define double_zero_is_octal 00
+C", q"D
+extern (C):
+
+enum some_octal_number = octal!101;
```

Comment:
Why is there no import for `octal` in this test?

Reply:
Good point!

Reply:
Fixed.